### PR TITLE
Adopt the PSF CoC

### PR DESCRIPTION
**Pull request by pradyunsg**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Code of Conduct
+
+Please note that all interactions related to Junction are covered by
+the [PSF Code of Conduct](https://www.python.org/psf/codeofconduct/),
```
Comment (bravegnu) [inline CODE_OF_CONDUCT.md]: Shouldn't this be the PyCon India CoC?

Comment (pradyunsg) [reply CODE_OF_CONDUCT.md]: PyCon India CoC is geared more towards a conference-only CoC, given the language.

PSF’s CoC is much more relaxed and general.

**Comment by pradyunsg** [comment]
Dependent on #576 being merged with CI fixes.

**Comment by zerothabhishek** [comment]
I think its okay to merge in spite of the minor reduction in coverage because this PR doesn't contain code.

**Pull request merged**